index.js

```diff
@@ -64,31 +54,43 @@ var FocusTrap = React.createClass({
     } else if (!prevProps.paused && this.props.paused) {
       this.focusTrap.pause();
     }
-  },
+  }
 
-  componentWillUnmount: function() {
+  componentWillUnmount() {
     this.focusTrap.deactivate();
     if (this.props.focusTrapOptions.returnFocusOnDeactivate !== false && this.previouslyFocusedElement) {
       this.previouslyFocusedElement.focus();
     }
-  },
+  }
 
-  render: function() {
-    var props = this.props;
+  render() {
+    const props = this.props;
```

Comment:
@DimitarChristoff Thank you for the comment. This code is unchanged from the base code. I was focussing on making it play nice with React 15.5 as this is the only package that is still warning on the console in our project. So I limited my refactor to the lines required to get this done.

I'd like @davidtheclark 's view on this first before changing more code as the widget has proven to be quite stable :)